### DISCUSSION TITLE
Vulkan Pro

### DISCUSSION THREAD

**Discussion by guybrushdent** [X3n0m0rph59/eruption#205] [Q&A]
So I have a Vulkan Pro in full size and noticed that some profiles show a blank line through the middle of my keys and shockwaves are activated under the wrong keys - what do you need me to do to implement this device fully? 
"eruption-util record-key-indices ev-dev" and then?

Great work btw - I'm running the "solid color" atm, but it's better than no light at all :+1:

**Comment by wormbones** [comment]
Hey, what distro do you have/installed it on? Wondering if I could ask you a couple questions.

**Reply by guybrushdent** [reply]
Just a plain Ubuntu with a 6.1 Kernel - nothing special tbh
